admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/screen/deposit/component/CountryRowItem.kt

```diff
@@ -0,0 +1,93 @@
+package net.thechance.mena.admin_panel.presentation.screen.deposit.component
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import net.thechance.mena.admin_panel.presentation.screen.deposit.DepositScreenState
+import net.thechance.mena.admin_panel.resources.Res
+import net.thechance.mena.admin_panel.resources.twemoji_mozilla
+import net.thechance.mena.designsystem.presentation.component.button.radioButton.RadioButton
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import org.jetbrains.compose.resources.Font
+
+@Composable
+internal fun CountryRowItem(
+    selectedCountry: DepositScreenState.CountryUiState,
+    isSelected: Boolean,
+    modifier: Modifier = Modifier,
+    onClick: (DepositScreenState.CountryUiState) -> Unit = {}
+) {
+
+    val animatedCountryItemColor by animateColorAsState(
+        targetValue = if (isSelected) Theme.colorScheme.background.surfaceHigh
+        else Theme.colorScheme.background.surfaceLow
+    )
```

Comment:
remove the if condition outside the function call
```suggestion
    val countryItemColor =  if (isSelected) Theme.colorScheme.background.surfaceHigh
        else Theme.colorScheme.background.surfaceLow
        
    val animatedCountryItemColor by animateColorAsState(
        targetValue = countryItemColor
    )
```